src/cplscheme/config/CouplingSchemeConfiguration.hpp

```diff
@@ -278,7 +278,7 @@ class CouplingSchemeConfiguration : public xml::XMLTag::Listener {
       DataID dataID) const;
 
   /// Get waveform order for read data of given name. Raises an error if data name is not found in the read data of this participant.
-  int getWaveformUsedOrder(std::string participantName, std::string readDataName) const;
+  // int getWaveformUsedOrder(std::string participantName, std::string readDataName) const;
```

Comment:
TODO?

Reply:
I think we don't need this function. Feel free to remove it.